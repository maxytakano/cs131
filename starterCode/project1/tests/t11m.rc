float arr[10];
bool a;
float myArr[10][11];

function : void foo(float &a)
{
  a[1] = a[9];
}

function : void main()
{
  foo(myArr);
}
