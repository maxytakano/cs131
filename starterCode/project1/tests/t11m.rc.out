Error, "/tmp/cs131_testrunner_cGTaux/file.rc", line 7: 
  Type of expression referenced by array subscript (float) is not of array or pointer type.
Error, "/tmp/cs131_testrunner_cGTaux/file.rc", line 7: 
  Type of expression referenced by array subscript (float) is not of array or pointer type.
Error, "/tmp/cs131_testrunner_cGTaux/file.rc", line 12: 
  Argument of type float[10][11] not equivalent to reference parameter a, of type float.
Compile: failure.
